support-frontend/assets/pages/[countryGroupId]/guardianLightLanding/components/posterComponent.tsx

```diff
@@ -1,3 +1,123 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	headlineBold24,
+	headlineBold34,
+	palette,
+	space,
+	textSans17,
+	textSans20,
+} from '@guardian/source/foundations';
+import { Container } from 'components/layout/container';
+
+const container = css`
+	background-color: ${palette.neutral[97]};
+	> div {
+		min-height: 480px;
+		padding: ${space[5]}px 10px;
+		${from.mobileLandscape} {
+			padding-left: ${space[5]}px;
+			padding-right: ${space[5]}px;
+		}
+		${from.tablet} {
+			min-height: 432px;
+			max-width: 740px;
+		}
+		${from.desktop} {
+			max-width: 940px;
```

Comment:
Possibly a refactor to consider for after this PR is in, but if dimensions such as 740px / 940px are used in Guardian Light page structure and components it may be worth putting them into variables that can be imported/shared across these files?

Reply:
Ok, will look afterwards in follow-up PR when doing final tweaks.